jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java

```diff
@@ -6,10 +6,35 @@
 
 import java.util.List;
 
+/**
+ * A Concrete fast Implementation for the interface #{@link Action},
+ * used to create a subset of actions gathered from the extracted actions of the assigned tween.
+ *
+ * Example of operation :
+ * <pre class="prettyprint">
+ * //create a base action from a tween.
+ * final BaseAction action = new BaseAction(Tweens.parallel(clipAction0, clipAction1));
+ * //set the action properties - utilized within the #{@link Action} class.
```

Comment:
What does the example actually do?

Reply:
@stephengold this example is used in production code, would i remove it too ? 

This is the github version : 
https://github.com/Scrappers-glitch/DBTraining/blob/06e968a5fc94d16f577629b869193c392083e9e0/app/src/main/java/com/scrappers/dbtraining/mainScreens/prefaceScreen/renderer/animationWrapper/EmitterTween.java#L310

Reply:
Please provide a more specific comment here. Something like "The following example runs two clips in sequence, at double speed, stopping after 5 seconds."

Reply:
If you think that the new change is good and clarify things, please resolve this conservation and continue reviewing.

Thanks.